tests/framework/microvm.py

```diff
@@ -370,6 +370,13 @@ def log_data(self):
             return ""
         return self.log_file.read_text()
 
+    @property
+    def console_data(self):
+        """Return the output of microVM's console"""
+        if self.screen_log is None:
+            return ""
```

Comment:
nit: would it be worth to differentiate it from the case where it exists but the output is empty (for example if the boot is hanging).

Reply:
It definitely would be nice to be able to do that, but I can't think of a way. Is there a way to check at this point if the microVM is actually running?

Reply:
oh, I see what you mean. Is there an Option type in python? :P

Reply:
Ok, I have it return `None` if the screen log is not there.